numba/parfor.py

```diff
@@ -246,20 +239,20 @@ def _convert_numpy(self, blocks):
         for label in topo_order:
             block = blocks[label]
             new_body = []
+            equiv_set = self.array_analysis.get_equiv_set(label)
             for instr in block.body:
                 if isinstance(instr, ir.Assign):
                     expr = instr.value
                     lhs = instr.target
-                    # only translate C order since we can't allocate F
-                    if self._has_known_shape(
-                            lhs) and self._is_C_order(lhs.name):
-                        if self._is_supported_npycall(expr):
-                            instr = self._numpy_to_parfor(lhs, expr)
+                    if self._is_C_order(lhs.name):
+                        # only translate C order since we can't allocate F
```

Comment:
I guess heavy linalg users would be the main group that would benefit as the LAPACK bindings assume 'F'. I suppose also anyone mixing with Fortran. 
@ehsantn Any idea of the level of effort to add support?
@seibert Do you have a view?